src/imgui/ImGuiSymbols.cc

```diff
@@ -158,11 +160,70 @@ static void drawTable(ImGuiManager& manager, const SymbolManager& symbolManager,
 
 			if (ImGui::TableNextColumn()) { // name
 				im::ScopedFont sf(manager.fontMono);
-				ImGui::TextUnformatted(sym.name(symbolManager));
+				ImGui::Selectable(sym.name(symbolManager).data());
 			}
 			if (ImGui::TableNextColumn()) { // value
 				im::ScopedFont sf(manager.fontMono);
-				ImGui::StrCat(hex_string<4>(sym.value(symbolManager)));
+				ImGui::Selectable(tmpStrCat(hex_string<4>(sym.value(symbolManager))).c_str());
+			}
+			if (ImGui::TableNextColumn()) { // slots
+				if (sym.slots(symbolManager) == -1) {
+					im::ScopedFont sf(manager.fontMono);
+					ImGui::Selectable("all slots");
+				} else {
+					zstring_view slots{};
+					std::string_view sep{};
+					for (auto sl = 0; sl < 4; ++sl) {
+						slots = tmpStrCat(slots, sep, sl);
```

Comment:
This is a bug:
* A `string_view` or `zstring_view` is only a view into another string that exists somewhere in memory.
* The function `tmpStrCat()` creates a `TemporaryString` object. A `(z)string_view` can indeed point to such an `TemporaryString` object. But that object is not kept alive (e.g. it's not kept alive in some local variable) so when this statement ends that object gets out of scope and gets deleted. And then the `(z)string_view` points to an invalid (deleted) object. It may only appear to work fine because in this specific case (with this version of the compiler) the memory that hold the `TemporaryString` happens to be not yet reused to hold something else.

One solution could be to change the type of `slots` from `zstring_view` to `std::string`. That type really "owns" the string-content (it does not merely refer to some existing string).

Instead of using `tmpStrCat()` (which returns a `TemporaryString`) it might be better to use plain `strCat()` which returns a `std::string`. And also instead of doing `x = strCat(x, ...)` it's more efficient to do `strAppend(x, ...)`.